src/features/completionItemProvider.ts

```diff
@@ -43,7 +43,7 @@ export default class OmniSharpCompletionItemProvider extends AbstractSupport imp
         try {
             let responses = await serverUtils.autoComplete(this._server, req);
```

Comment:
If you look at the defintion of serverUtils.autoComplete, it calls server.MakeRequest, which supports passing though a CancellationToken. Interested in threading that through so you can pass it here?

Reply:
Ah yea that makes total sense. Will take a look.